pkg/webui/console/containers/application-overview-header/index.js

```diff
@@ -79,6 +96,10 @@ const ApplicationOverviewHeader = () => {
   const deleteBookmarkLoading = useSelector(state =>
     selectFetchingEntry(state, DELETE_BOOKMARK_BASE),
   )
+  const webhooksCount = useSelector(selectWebhooksTotalCount)
+  const pubsubsCount = useSelector(selectPubsubsTotalCount)
+  const hasIntegrations = webhooksCount > 0 || pubsubsCount > 0
+  const additionalConditions = hasIntegrations
```

Comment:
```suggestion
  const additionalCondition = hasIntegrations
```
Nit: plural makes it sound like it expects an array here.